lib/transforms/subsetGoogleFonts.js

```diff
@@ -133,11 +133,20 @@ var fontFormatUA = {
     woff2: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.96 Safari/537.36'
 };
 
+var validFontDisplayValues = [
+    'auto',
+    'block',
+    'swap',
+    'fallback',
+    'optional'
+];
+
 module.exports = function (options) {
     options = options || {};
     var subsetPerPage = !!options.subsetPerPage;
     var inlineSubsets = !!options.inlineSubsets;
     var inlineCss = !!options.inlineCss;
+    var fontDisplay = validFontDisplayValues.includes(options.fontDisplay) && options.fontDisplay;
```

Comment:
The build fails with node 4 due to the use of `Array#includes`

Reply:
Fixed and rebased